tests/sles4sap/ensa/netweaver_ensa2_web_methods.pm

```diff
@@ -61,6 +73,11 @@ sub run {
     $self->webmethod_checks($instance_id);
     # Execute failover from ASCS instance - this will return resources to original host
     if ($instance_type eq 'ASCS') {
+        # Some delay needed here for 12-SP5, don't know why even webmethod_checks passed
+        if (is_sle('=12-SP5')) {
+            sleep 300;
```

Comment:
Is the 300s needed here too even after waiting for up to 300s in `$self->webmethod_checks`?